core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -714,6 +720,12 @@ object KafkaConfig {
   val ShareGroupAssignorsDoc = "The server-side assignors as a list of full class names. The first one in the list is considered as the default assignor to be used in the case where the share group does not specify an assignor."
   val ShareGroupPersisterClassNameDoc = "The class name of share persister for share group. The class should implement " +
     "the <code>org.apache.kafka.server.group.share.Persister</code> interface."
+  val ShareCoordinatorStateTopicPartitionsDoc = "Number of partitions for the internal share-group state topic."
+  val ShareCoordinatorStateTopicReplicationFactorDoc = "Replication factor for the share-group state internal topic. " +
+  "Internal topic creation will fail until the cluster size meets this replication factor requirement."
+  val ShareCoordinatorStateTopicSegmentBytesDoc = "The share-group state topic segment bytes should be kept relatively small in order to facilitate faster log compaction and cache loads."
```

Comment:
I suggest just using the text from the KIP. In this case, this is not a compacted topic, so the point about log compaction doesn't apply. `"The log segment size for the share-group state topic."`